File: nodejs/mysql.js
var mysql = require('mysql2');

var conn = mysql.createConnection({
  host: "[HOSTNAME]",
  user: "[USERNAME]",
  password: "[PASSWORD]",
  database: "[DATABASE]",
  ssl: {
    rejectUnauthorized: true
  }
});

conn.connect(function(err) {
  if (err) throw err;
  console.log("Succesfully connected to PlanetScale!");
  process.exit(0)
});

File: java/src/main/java/com/planetscale/example/mysql.java
package com.planetscale.example;

import java.sql.*;

class Mysql {
    public static void main(String args[]){
        try {
            Class.forName("com.mysql.cj.jdbc.Driver");
            Connection conn = DriverManager.getConnection(
                "jdbc:mysql://[HOSTNAME]/[DATABASE]?sslMode=VERIFY_IDENTITY",
                "[USERNAME]",
                "[PASSWORD]");
            conn.close();
            System.out.println("Successfully connected to PlanetScale!");
        } catch(Exception e) {
            e.printStackTrace();
            System.exit(1);
        }
    }
}
